domains/rntxpurval.is-a-fullstack.dev.json

```diff
@@ -0,0 +1,20 @@
+{
+    "description": "For Hosting My projects",
+    "domain": "is-a-fullstack.dev",
+    "subdomain": "rntxpurval",
+
+    "owner": {
+        "repo": "https://github.com/purval26/food-del",
+        "email": "purvalradadiya@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "tomato.rntxpurval.is-a-fullstack.dev",
```

Comment:
CNAME records cannot be used in conjunction with other records.

Reply:
i cant understand